src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -52,6 +53,10 @@ function useComputeSwapInfo(): SwapInfo {
     useMemo(() => [inputCurrency ?? undefined, outputCurrency ?? undefined], [inputCurrency, outputCurrency])
   )
 
+  // Short circuit trade fetching and amount parsing if wrapping or unwrapping.
```

Comment:
If you're short-circuiting here, can you remove the short-circuiting in useClientSideSmartOrderRouterTrade? Or are they both necessary?

Reply:
I actually didnt know we did that in `useClientSideSmartOrderRouterTrade`, ill remove here instead